digiwf-libs/digiwf-camunda-prometheus/digiwf-camunda-prometheus-starter/src/main/java/de/muenchen/oss/digiwf/camunda/prometheus/CamundaPrometheusProperties.java

```diff
@@ -0,0 +1,38 @@
+package de.muenchen.oss.digiwf.camunda.prometheus;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "digiwf.prometheus.process-engine")
+public class CamundaPrometheusProperties {
+    /**
+     * Update interval for running metrics providers, which actively execute queries.
+     */
+    private int updateInterval;
```

Comment:
```suggestion
    private int updateInterval = 30000;
```

Würde den Default jetzt eher über die Config setzen.